README.md

```diff
@@ -1,128 +1,78 @@
-
-This section is intended to provide readers of your pull request with a fuller picture of the proposed feature by comparing it with similar features in other tools and offering visual aids. If you don't have any examples of similar features being implemented elsewhere, that is fine—your ideas are interesting to us, whether they are brand new or adaptations from other tools.
-
-#### IV. Explanation
-
-Thoroughly explain your feature as if it was already implemented in Meilisearch and you were teaching another user how to use it. That generally means:
-
-- Introducing (and naming) any new concepts.
-- Explaining the feature largely through examples.
-- Noting the API for this feature, HTTP, CLI or config.
-- Explaining how the user should _think_ about the feature and how it will impact the way they use Meilisearch (provide concrete examples).
-- If applicable, provide sample error messages, deprecation warnings, or migration guidance.
-
-If the changes modify the HTTP API, provide a description of the method, URL, parameters, body, status code, errors, etc...
+---
 
-If it modifies the CLI, provide the env variable name, the argument name, and the description.
+## Specification File Format
 
-This section serves as a user-level guide and should resemble official documentation. Anything that the user may encounter while interacting with the feature should be presented here.
+Meilisearch's feature specifications are made up of five sections, described below.
 
-#### V. Impact on Documentation
+### 1. Summary
 
-If the feature requires additions or updates to the documentation, they should be noted here. It's the role of the documentation team to ensure this section of the feature specification is accurate.
+Summarize the specification with a short paragraph.
 
-#### VI. Impact on SDKs
+### 2. Motivation
 
-If the feature requires additions or updates to the SDks, they should be noted here. It's the role of the integration team to ensure this section of the feature specification is accurate.
+Explains what use cases does it support.
 
-### 2. Technical Aspects
+### 3. Functional Specification
 
-This section of the feature specification document has a much narrower audience: the developer(s) that will implement the feature. Its goal is to discuss and clarify how the feature will be developed and implemented when its not a trivial concern.
+This section gives a high level overview of the feature. It should avoid technical language so that it can be understood by a general audience (think user-level).
 
-> Internal technical teams may use this part to keep track of technical brainstorms, related proto code or basecode examples to clear the technical details of the specification.
+- Describes the API resource and endpoints. (Methods, URL, query parameters, body definition, status code).
+- Explains the feature through examples.
+- Lists error cases.
```

Comment:
```suggestion
- Describe the API resource and endpoints. (Methods, URL, query parameters, body definition, status code).
- Explain the feature through examples.
- List error cases.
```

Reply:
For me you are talking to the person reading the process, so you give information about what to do